sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/models/KeyReleasePolicy.java

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.security.keyvault.keys.models;
+
+import com.azure.core.annotation.Fluent;
+import com.azure.core.util.Base64Url;
+import com.azure.core.util.CoreUtils;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * A model that represents the policy rules under which the key can be exported.
+ */
+@Fluent
+public final class KeyReleasePolicy {
+    /*
+     * Content type and version of key release policy.
+     */
+    @JsonProperty(value = "contentType")
+    private String contentType;
+
+    /*
+     * Blob encoding the policy rules under which the key can be released.
+     */
+    @JsonProperty(value = "data")
+    private Base64Url data;
+
+    /**
+     * Get the content type and version of key release policy.
+     *
+     * @return The content type and version of key release policy.
+     */
+    public String getContentType() {
+        return this.contentType;
+    }
+
+    /**
+     * Set the content type and version of key release policy.
+     *
+     * <p>The service default is "application/json; charset=utf-8".</p>
+     *
+     * @param contentType The content type and version of key release policy to set.
+     *
+     * @return The updated {@link KeyReleasePolicy} object.
+     */
+    public KeyReleasePolicy setContentType(String contentType) {
+        this.contentType = contentType;
+
+        return this;
+    }
+
+    /**
+     * Get a blob encoding the policy rules under which the key can be released.
+     *
+     * @return A blob encoding the policy rules under which the key can be released.
+     */
+    public byte[] getData() {
+        if (this.data == null) {
+            return null;
+        }
+
+        return this.data.decodedBytes();
```

Comment:
Why decode the bytes every time instead of once and return the reference? Is providing immutability required for the Java SDKs? We don't attempt to do that in .NET at least. I don't see anything in Java's guidelines apart from immutable service clients - nothing about response models.

Reply:
This is what all classes generated by AutoRest do, so I thought I would follow that pattern. That said, I don't think it's a guideline and we can probably get away without decoding every time. I'll bring it up with our Java architect/deputy architect.

Reply:
Now that I think about it, I think we do this to ensure the payload is properly encoded when sending it through the wire. I will still discuss this with them though.

Reply:
Just heard back: we do this when the swagger specification marks a field as `base64url`. I don't think it should be much of a performance issue, I wouldn't expect people to access this field an overwhelming amount of times.

Reply:
FYI, we have other properties in Keys defined as `base64Url` in the swagger that are actually saved as a byte array, [like those that are a part of `JsonWebKey`](https://github.com/Azure/azure-sdk-for-java/blob/cbba4149f87978ba16c0696ebe7177e6238c4598/sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/models/JsonWebKey.java#L235), however, we still clone them when setting/getting them.

Reply:
Ended up using a byte array type for this variable and using Jackson to properly serialize/deserialize it as base64.